feat(cli): Add --ignore-missing flag documentation (#3625)
getsentry/sentry-docs: src/docs/product/cli/releases.mdx

You can use the `--local` flag to enable this behavior by default.

```bash
sentry-cli releases set-commits "$VERSION" --local
```

If you receive an "Unable to Fetch Commits" email, take a look at our [Help Center Article](https://help.sentry.io/hc/en-us/articles/360019866834-Why-am-I-receiving-the-email-Unable-to-Fetch-Commits-).

### Dealing With Missing Commits

There are scenarios in which your repositories may be missing commits previously used in the release.

> It can happen whenever you modify the commit in question, for example by ammending it, rebasing or squashing multiple commits together.

Reviewer: ```suggestion
This can happen whenever you modify the commit in question by, for example, amending it, rebasing, or squashing multiple commits together.
```

> In this case, Sentry CLI will be unable to find it, and will throw an error explaining what happened.

Reviewer 2: Just to clarify, since I believe the CLI throws an error, but doesn't necessarily explain anything other than that the CLI can't find the commit.
```suggestion
In this case, Sentry CLI will be unable to find it, and will throw an error that the commit cannot be found.
```

> When this happens, you can pass an additional `--ignore-missing` flag. This will allow the command to fall back to the default behavior,

Reviewer 2: ```suggestion
If Sentry CLI throws this error, you can pass an additional `--ignore-missing` flag, which will allow the command to fall back to the default behavior,
```

> Whenever this happens, you can pass an additional `--ignore-missing` flag, which will allow the command to fall back to the default behavior,

Reviewer: ```suggestion
When this happens, you can pass an additional `--ignore-missing` flag. This will allow the command to fall back to the default behavior,
```
